piccolo_api/csrf/middleware.py

```diff
@@ -120,6 +120,8 @@ async def dispatch(
                     self.cookie_name,
                     token,
                     max_age=self.max_age,
+                    httponly=True,
+                    secure=True,
```

Comment:
The only reason this was currently set to `False` is because when using Piccolo Admin locally, most people wouldn't run it under HTTPS so it would stop working. I think a good compromise would be to have this configurable. For example, we have a `production` argument with Session auth:

https://github.com/piccolo-orm/piccolo_api/blob/f3e17a610a61c2cbedf57cf154a339f85ba8d84e/piccolo_api/session_auth/endpoints.py#L330

Reply:
Hmm, this needs to be updated in piccolo admin as well, right?

Reply:
Yeah, that's right. We would pass `production` in here: https://github.com/piccolo-orm/piccolo_admin/blob/5e8e96d56656cbf7283b63f8729d0748eb895867/piccolo_admin/endpoints.py#L439